src/native/libs/System.Security.Cryptography.Native.Android/pal_jni.c

```diff
@@ -704,6 +707,8 @@ JNI_OnLoad(JavaVM *vm, void *reserved)
     g_ivPsClass =               GetClassGRef(env, "javax/crypto/spec/IvParameterSpec");
     g_ivPsCtor =                GetMethod(env, false, g_ivPsClass, "<init>", "([B)V");
 
+    g_AEADBadTagExceptionClass = GetClassGRef(env, "javax/crypto/spec/AEADBadTagException");
```

Comment:
```suggestion
    g_AEADBadTagExceptionClass = GetClassGRef(env, "javax/crypto/AEADBadTagException");
```

The name of the class seems incorrect.

Reply:
Ohh thank you!